src/components/NavBar.js

```diff
@@ -2,7 +2,7 @@ import { NavLink as RouterLink } from 'react-router-dom';
 
 export default function NavBar(props) {
   return (
-    <nav className="grid grid-cols-2 bg-blue-ncs p-5 text-lg md:text-xl w-full divide-x-2 divide-gray-200 absolute bottom-0 md:px-36 lg:px-48 xl:px-72 lg:w-2/3">
+    <nav className="grid grid-cols-2 bg-blue-ncs p-5 text-lg md:text-xl w-full divide-x-2 divide-gray-200 fixed bottom-0 z-10 md:px-36 lg:px-48 xl:px-72 lg:w-2/3">
```

Comment:
Ahhhh it just needed that z in there with the fixed. Nice!

Reply:
Yeah, and that <div> with the margin bottom was crucial, or the bottom of the list would still be hidden!